### PR TITLE
Add memory profiler

### PR DESCRIPTION
**Pull request by vlad9486**

### CODE REVIEW

File: apps/deploy_monitoring/src/image.rs [hunk 1]
```diff
@@ -76,3 +76,9 @@ pub struct Sandbox;
 impl DeployMonitoringContainer for Sandbox {
     const NAME: &'static str = "deploy-monitoring-tezedge-sandbox-launcher";
 }
+
+pub struct TezedgeMemprof;
```
Comment (bkontur) [inline apps/deploy_monitoring/src/image.rs:80]: @vlad9486 @adonagy @jurajselep 
do we want this just for TezEdge node or also for Ocaml node?

File: apps/deploy_monitoring/docker-compose.deploy.latest.yml [hunk 2]
```diff
@@ -1,6 +1,18 @@
 version: "3"
 
 services:
+  tezedge-memprof:
```
Comment (bkontur) [inline apps/deploy_monitoring/docker-compose.deploy.latest.yml:4]: @vlad9486 @adonagy @jurajselep 
do we want to run memprof also on master.dev.tezedge.com?
if so, we need to add this also to `docker-compose.deploy.latest-release.yml`  right?

File: apps/deploy_monitoring/src/image.rs (hunk 1 above)
Comment (vlad9486) [reply apps/deploy_monitoring/src/image.rs:80]: @bkontur Now it work only for tezedge node. The name 'light-node' hardcoded into bpf module.

File: apps/deploy_monitoring/docker-compose.deploy.latest.yml (hunk 2 above)
Comment (vlad9486) [reply apps/deploy_monitoring/docker-compose.deploy.latest.yml:4]: @bkontur You are right we will run it on master. I'll add.

**Comment by bkontur** [comment]
@vlad9486 
could you please remove `libflate` dependency from deploy_monitoring/Cargo.toml,
it looks like it is unused

**Review by bkontur — APPROVED** [review]
@vlad9486 
for me ok

@adonagy 
Ado, what about you?

@vlad9486 
if Ado confirms, then rebase/squash/merge to develop, please

**Review by adonagy — APPROVED** [review]
Looks good to me

**Pull request merged**